src/cli/cli.cpp

```diff
@@ -5700,6 +5774,35 @@ template <> otError Interpreter::Process<Cmd("panid")>(Arg aArgs[])
     return error;
 }
 
+/**
+ * @cli parent
+ * @code
+ * parent
+ * Ext Addr: be1857c6c21dce55
+ * Rloc: 5c00
+ * Link Quality In: 3
+ * Link Quality Out: 3
+ * Age: 20
+ * Version: 4
+ * Done
+ * @endcode
+ * @code
+ * CSL clock accuracy: 20
+ * CSL uncertainty: 5
+ * @endcode
+ * @par api_copy
+ * #
+ * @par
+ * Get the diagnostic information for a Thread Router as parent.
+ * @par
+ * Note: When operating as a Thread Router when OPENTHREAD_CONFIG_REFERENCE_DEVICE_ENABLE is enabled, this command
```

Comment:
Remove the Note: from this, just leave it as a normal paragraph, it doesn't look good for all the text to be a note on the website